Amplify/Core/Category/CategoryType.swift

```diff
@@ -62,3 +62,26 @@ public extension CategoryType {
         }
     }
 }
+
+public extension CategoryType {
```

Comment:
there's already `public extension CategoryType` at line 43, you can just add `var category` under `var displayName`